scripts/zkai_d64_stwo_vector_row_surface_probe.py

```
@@ -0,0 +1,523 @@
+    except ValueError as err:
+        raise D64VectorRowSurfaceError("generated_at must be a valid UTC timestamp string") from err
+    if parsed.tzinfo != dt.timezone.utc:
+        raise D64VectorRowSurfaceError("generated_at must be a UTC timestamp string")
+
+
+def _git_commit() -> str:
+    override = os.environ.get("ZKAI_D64_STWO_SURFACE_PROBE_GIT_COMMIT")
+    if override and override.strip():
+        return override.strip().lower()
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)